test/conformance/source/environment.cpp

```diff
@@ -100,11 +106,18 @@ uur::PlatformEnvironment::PlatformEnvironment(int argc, char **argv)
 
 void uur::PlatformEnvironment::SetUp() {
     if (!error.empty()) {
-        FAIL() << error;
+        if (error == "Could not load adapter") {
```

Comment:
nit - Using strings to differentiate between different error scenarios isn't ideal. For tests it's fine, but defining something like might make it cleaner:
```
constexpr std::string_view ERROR_NO_ADAPTER = "Could not load adapter";
...
if (error == ERROR_NO_ADAPTER) {
    ...
}
```

Reply:
No `std::string_view` on C++14 unfortunately, but will fix.